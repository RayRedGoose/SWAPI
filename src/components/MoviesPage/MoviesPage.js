import React from 'react';
import UserProfile from '../UserProfile/UserProfile';
import MoviesBox from '../MoviesBox/MoviesBox';
import './MoviesPage.scss';

const MoviesPage = () => {

  return (
    <section className="moviesPage">
      <UserProfile />
      <h2>Star Wars Movies</h2>
      <MoviesBox />
    </section>
  )
}

export default MoviesPage;
