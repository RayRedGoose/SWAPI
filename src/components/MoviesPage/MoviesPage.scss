.moviePage {
  background: papayawhip;
}

.userProfile {
  display: block;
  position: relative;
  float: right;
  top: auto;
  border: 2px solid black;
  padding: 3em 5em 3em 5em;
    section {
      display: none;
    }
}

h2 {
  padding-top: 1em;
  padding-left: .5em;
  padding-bottom: 1.5em;
  font-size: 3.5em;
}
