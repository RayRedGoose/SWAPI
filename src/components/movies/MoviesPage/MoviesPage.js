import React from 'react';
import UserProfile from '../../UserProfile/UserProfile';
import MoviesBox from '../MoviesBox/MoviesBox';
import './MoviesPage.scss';

const MoviesPage = (props) => {
  return (
    <section className="moviePage">
      <header>
        <h2>Star Wars Movies</h2>
        <UserProfile
          {...props.user}
          logOut={props.logOut}
         />
      </header>
      <MoviesBox
        {...props.sys}
        movies={props.movies}
        addMovies={props.addMovies}
      />
    </section>
  )
}

export default MoviesPage;
