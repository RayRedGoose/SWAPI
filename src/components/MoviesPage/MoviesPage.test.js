import React from 'react';
import { shallow } from 'enzyme';
import MoviesPage from './MoviesPage';

describe("MoviesPage", () => {

  it('should render MoviesPage', () => {
    const wrapper = shallow(
      <MoviesPage
      user={"The Rock"}
      logOut={jest.fn()}
      />)
    expect(wrapper).toMatchSnapshot();
  })
})
