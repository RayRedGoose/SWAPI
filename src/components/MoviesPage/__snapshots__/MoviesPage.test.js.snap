// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesPage should render MoviesPage 1`] = `
<section
  className="moviePage"
>
  <section
    className="userProfile"
  >
    <UserProfile
      0="T"
      1="h"
      2="e"
      3=" "
      4="R"
      5="o"
      6="c"
      7="k"
      logOut={[MockFunction]}
    />
  </section>
  <h2>
    Star Wars Movies
  </h2>
  <MovieBox />
</section>
`;
